plugins/QnA/class.qna.plugin.php

```diff
@@ -630,7 +637,7 @@ public function discussionController_QnA_create($sender, $args) {
                     case 'Accepted':
                         $Change = 1;
 
-                        if (c('QnA.Points.Enabled', false) && $Discussion['InsertUserID'] != $Comment['InsertUserID']) {
+                        if (!$this->Reactions && c('QnA.Points.Enabled', false) && $Discussion['InsertUserID'] != $Comment['InsertUserID']) {
```

Comment:
Won't this property be entirely missing and generate a notice if Reactions is off?